app/controllers/admin/runs_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class Admin::RunsController < Admin::BaseController
   def index
-    authorize :runs, :index?
+    authorize :run, :index?
```

Comment:
All these replaces in the `authorize` calls are because of the `bundle update` you did, right?